lib/blueprinter/base.rb

```diff
@@ -292,8 +293,8 @@ def self.fields(*field_names)
     #   end
     #
     # @return [Array<Symbol>] an array of view names.
-    def self.include_view(view_name)
-      current_view.include_view(view_name)
+    def self.include_view(view_name = nil)
+      current_view.include_view(view_name || Proc.new)
```

Comment:
Above this method, would you add comments to note that it optionally accepts a proc? As well as an example?
For example, `#field` has the following comments: https://github.com/procore/blueprinter/blob/4ba12dd14edf340f9a4a95f1668785adc8c21ca6/lib/blueprinter/base.rb#L87-L102

These comments are parsed by yard and uploaded to rubydocs which produces documentation like this: https://www.rubydoc.info/github/procore/blueprinter/Blueprinter/Base#field-class_method

Additionally, would you update the readme in the "View" section to give an example of passing a block to `include_view`